fontforge/plugin.c

```diff
@@ -445,24 +446,34 @@ void PyFF_ImportPlugins(int do_import) {
 }
 
 extern PyObject *PyFF_GetPluginInfo(PyObject *UNUSED(noself), PyObject *UNUSED(args)) {
-    PyObject *r, *t;
+    PyObject *r, *tmp, *entry;
     GList_Glib *l;
     PluginEntry *pe;
 
     r = PyList_New(0);
 
     for (l = plugin_data; l != NULL; l = l->next) {
         pe = (PluginEntry *) l->data;
-        t = Py_BuildValue("ssssssOss", pe->name,
-                          pluginStartupModeString(pe->startup_mode, false),
-                          pluginInfoString(pe, false, NULL),
-                          pe->package_name,
-                          pe->module_name,
-                          pe->attrs,
-                          pe->has_prefs ? Py_True : Py_False,
-                          pe->package_url,
-                          pe->summary);
-        PyList_Append(r, t);
+        entry = PyDict_New();
+	tmp = PyUnicode_FromString(pe->name);
```

Comment:
I think you need to `Py_DECREF` these. Might be easier to still use `Py_BuildValue`? [e.g.](https://docs.python.org/3.3/extending/extending.html#building-arbitrary-values)

```
Py_BuildValue("{s:i,s:i}",
              "abc", 123, "def", 456)    {'abc': 123, 'def': 456}
```

Reply:
OK ... done